modules/Bio/EnsEMBL/IO/Adaptor/HTSAdaptor.pm

```diff
@@ -153,10 +116,10 @@ sub fetch_paired_alignments {
 
   my @features;
 
-  my $header = $hts_obj->hts_file->header_read;
+  my $header = $hts_obj->header;
 
   # Maybe need to add 'chr'
-  my $seq_id = $self->munge_chr_id($chr_id);
+  my $seq_id = $self->munge_chr_id($hts_obj, $chr_id);
```

Comment:
would it be possible to invert the two elements?
I realise it will fail if either arguments are missing, but for sanity, I would rather add a new argument at the end of the list of existing arguments rather than mess with the order